tests/GistConfigTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+use Gistlog\Gists\GistConfig;
+
+class GistConfigTest extends TestCase
+{
+    use GistFixtureHelpers;
+
+    /** @test */
+    public function it_can_be_created_from_github_api_data()
+    {
+        $githubGist = $this->loadFixture('8f5ea4d44dbc5ccb77a3.json');
+
+        $config = GistConfig::fromGitHub($githubGist);
+
+        $this->assertEquals("A test preview.", $config['preview']);
+        $this->assertEquals("05-15-2015", $config['published_on']);
```

Comment:
Hm, if we're choosing any date format I would choose `YYYY-MM-DD` so it's globally clear.